src/routes/+layout.svelte

```diff
@@ -1,53 +1,50 @@
 <script>
 	import '../app.postcss';
 
-	import '@fontsource/inter/300.css';
-	import '@fontsource/inter/400.css';
-	import '@fontsource/inter/500.css';
-	import '@fontsource/inter/600.css';
-	import '@fontsource/inter/700.css';
-	import '@fontsource/inter/800.css';
-	import '@fontsource/inter/900.css';
-
-	import GithubLogo from '~icons/radix-icons/github-logo';
-	import DiscordLogo from '~icons/radix-icons/discord-logo';
-
 	import { dev } from '$app/environment';
 	import { inject } from '@vercel/analytics';
 
 	inject({ mode: dev ? 'development' : 'production' });
 </script>
 
 <main class="flex min-h-screen flex-col">
-	<nav class="flex items-center justify-between gap-2 border-b border-zinc-400 px-2 lg:px-4 py-2">
-		<a class="flex items-center gap-2 lg:text-xl font-sans font-bold" href="/">
+	<nav class="flex items-end justify-between px-6 py-4">
+		<a class="flex items-center gap-2 font-sans text-xl" href="/">
 			<img
-				class="w-7 h-7 lg:h-9 lg:w-9 rounded-sm object-contain"
-				src="/logo.svg"
+				class="h-8 w-8 rounded-sm object-contain"
+				src="/radix-svelte.svg"
```

Comment:
I liked the orange logo here 🤔

Reply:
I think the monochrome one looks sleeker tbh